doc/cpp/cable_cell.rst

```diff
@@ -8,6 +8,8 @@ Cable cells
    has changed significantly; some of the documentation below is
    out of date.
 
+   The C++ cable cell documentation should have the same structure as the Python cable cell documentation.
```

Comment:
Why does this need to go in the docs?

Reply:
Since there was already a comment about being out of date, I thought a hint as to what needs to happen wouldn't hurt. Can be removed if you prefer.